src/Compilers/Core/Portable/DiagnosticAnalyzer/CompilationWithAnalyzers.cs

```diff
@@ -1138,6 +1149,18 @@ private async Task<Task> SetActiveTreeAnalysisTaskAsync(Func<Tuple<Task, Cancell
             }
         }
 
+        private async Task WaitForExecutingTaskAsync(Task executingTask)
```

Comment:
>private [](start = 8, length = 7)

`static`

Reply:
Yes, sure. I can fix it in my other open PR which nullable enables analyzer driver.